cmd/util/cmd/remove-execution-fork/cmd/root.go

```diff
@@ -0,0 +1,37 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	flagDatadir string
+)
+
+// run with `./remove-execution-fork remove-execution-fork --datadir /var/flow/data/protocol`
```

Comment:
Is the idea to combine this with a larger cobra-based CLI utility? If not we can probably remove the sub-command and invoke the root command directly with `./remove-execution-fork`